challenges/linked-list/test_linked_list.py

```diff
@@ -0,0 +1,36 @@
+
+from linked_list import mergeLists
+
+# @pytest.fixture
+# def empty_linked_list():
+#     return LL()
+
+
+# @pytest.fixture
+# def n_linked_list_list():
+#     return LL([1, 2, 3])
+
+
+def test_mergeLists_xempty(empty_ll, n_linked_ll):
```

Comment:
You should have docstrings for your tests too. These are useful to give more details about the test that are not covered in the name of the function.